frontend/src/js/ui-components/InputMultiSelect/InputMultiSelect.tsx

```diff
@@ -264,6 +265,16 @@ const InputMultiSelect = ({
       ? filteredOptions.length - 1
       : filteredOptions.length;
 
+  // scroll option list into view if neccessary
+  useEffect(
+    function scrollIntoView() {
+      if (isOpen) {
+        menuRef.current?.scrollIntoView({ behavior: 'smooth', block: 'nearest' })
```

Comment:
Just do double-check, did you configure code-formatting (run prettier on saving a file in your editor)? For me this formats differently (--> I will add a CI step for format lint soon)